library/ssl_misc.h

```diff
@@ -802,6 +806,11 @@ struct mbedtls_ssl_handshake_params
                                              represents an extension and defined
                                              as \c MBEDTLS_SSL_EXT_XXX */
 
+#if defined(MBEDTLS_KEY_EXCHANGE_WITH_CERT_ENABLED)
+    unsigned char certificate_request_context_len;
+    unsigned char *certificate_request_context;
+#endif
```

Comment:
```suggestion
```
It is not used yet

Reply:
I agree, it is not used currently because we do not send the client Certificate message yet. But that is expected and the support for the client Certificate message is coming thus that's ok to keep it in this PR.

Reply:
Got it